src/mame/apple/apple2e.cpp

```diff
@@ -1165,6 +1165,14 @@ void apple2e_state::machine_reset()
 	m_xy = false;
 	m_x0edge = false;
 	m_y0edge = false;
+	last_mx = 0;
+	last_my = 0;
+	count_x = 0;
+	count_y = 0;
+	m_x0 = false;
+	m_y0 = false;
+	m_x1 = false;
+	m_y1 = false;
```

Comment:
You shouldn’t be changing these on reset as they’re not registers affected by reset.  They should only be cleared on start.

Reply:
I can move the first 6 from machine_reset to machine_start.  For the last 2, X1/Y1 are returned by soft switches C066, C067, C06E, C06F.  Should they be reset in this case?  If not, I will move those as well.